src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -273,6 +277,28 @@ def update_azure_storage_account(cmd, resource_group_name, name, custom_id, stor
     return result.properties
 
 
+def enable_zip_deploy_functionapp(cmd, resource_group_name, name, src, timeout=None, slot=None):
+    client = web_client_factory(cmd.cli_ctx)
+    app = client.web_apps.get(resource_group_name, name)
+    parse_plan_id = parse_resource_id(app.server_farm_id)
+    plan_info = None
+    retries = 5
+    retry_delay = 10  # seconds
+    # We need to retry getting the plan because sometimes if the plan is created as part of function app,
+    # it can take a couple of tries before it gets the plan
+    while retries > 0:
```

Comment:
In Python, we usually do `for _ in range(5)` and avoid the temporary variable of `retries`